Services/Utilities/classes/class.ilUtil.php

```diff
@@ -1817,7 +1817,16 @@ public static function unzip($a_file, $overwrite = false, $a_flat = false)
 		ilUtil::execQuoted($unzip, $unzipcmd);
 
 		chdir($cdir);
-		
+
+		// remove all sym links
+		foreach (new RecursiveIteratorIterator(new RecursiveDirectoryIterator($dir)) as $name => $f)
+		{
+			if (is_link($name))
+			{
+				unlink($name);
```

Comment:
This is why I asked for any use cases for symlinks in the bug report at all. End users will not have access to the log.